cumulus/pallets/xcmp-queue/src/lib.rs

```diff
@@ -132,6 +131,25 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxInboundSuspended: Get<u32>;
 
+		/// Maximal number of outbound XCMP channels that can have messages queued at the same time.
+		///
+		/// If this is reached, then no further messages can be sent to channels that do not yet
+		/// have a message queued. This should be set to the expected maximum of outbound channels
+		/// which is determined by [`Self::ChannelInfo`]. It is important to set this correctly,
```

Comment:
What does "correctly" mean in this case? Is there any other information we could add here to make it better for users setting this?

Reply:
It can go wrong when a chain has like 200 channels open but they set this value to 100.  
The migration does prevent this in the `pre_upgrade` hook, so it should be mostly fine. Unless opengov decides to open a channel right before the RA.  
Will add that as comment.